designer/server/src/main/scala/pl/touk/nussknacker/ui/api/DictResources.scala

```diff
@@ -0,0 +1,29 @@
+package pl.touk.nussknacker.ui.api
+
+import akka.http.scaladsl.marshalling.ToResponseMarshallable
+import akka.http.scaladsl.model.{HttpResponse, StatusCodes}
+import akka.http.scaladsl.server.{Directives, Route}
+import de.heikoseeberger.akkahttpcirce.FailFastCirceSupport
+import pl.touk.nussknacker.engine.api.dict.DictQueryService
+
+import scala.concurrent.ExecutionContext
+
+class DictResources(implicit ec: ExecutionContext) extends Directives with FailFastCirceSupport {
```

Comment:
Sorry for the late announcement but we create every new endpoint in tapir. Can you rewrite it to tapir?

Reply:
will do in separate PR